custom_components/myskoda/sensor.py

```diff
@@ -176,9 +176,11 @@ class RemainingDistance(ChargingSensor):
 
     @property
     def native_value(self) -> int | float | None:  # noqa: D102
-        if status := self._status():
-            if status.battery.remaining_cruising_range_in_meters is not None:
-                return status.battery.remaining_cruising_range_in_meters / 1000
+        if range := self.vehicle.driving_range:
```

Comment:
nit: maybe we should rename this class to `Range` now? And update the doc-string accordingly.

Reply:
Also its not a charging sensor any more, chargingsensors only show up on cars with charging capability